Vagrantfile

```diff
@@ -97,6 +97,11 @@ if File.exist?(env_specific_config_path)
   end
 end
 
+# Display the main current configuration parameters
+@ui.info "Welcome to Kubernetes playground !"
+@ui.info "Vagrant provider : " + settings["conf"]["vagrant_provider"]
```

Comment:
1) `Currently selected Vagrant provider:`
2) I think we defined and initialized a `vagrant_provider` variable somewhere in the `Vagrantfile`, we should use that.

Reply:
it's defined later... I've considered this issue, I have a small preference to output here exactly the raw values that are in the setting, and this is the logical place where to output it... even before we do the checks... 
yes, there is a cost of reading twice some parameters from settings but i would pay it so that we have a clean and simple approach: when we want to output some value to the ui, we just go in this place, we read and write the value
do you have a strong preference for the other approach?

Reply:
I got what you say.

What if we output the whole configuration in a compact form? With a trick, you can have it as a nicely formatted YAML `settings.to_yaml` and print that, so we take the raw values as they are, and we don't have to bother about adding more stuff to this output when we add parameters.

Note that the `.to_yaml` method outputs, well, YAML. This might clutter things a bit, so we might have to find a more compact representation.

Reply:
IMO the full configuration file is overkill for the normal usage... but this is my opinion...
in this case I prefer to have three lines that gives me the important information
we can open a issue to discuss further